src/Microsoft.Data.Analysis.IO/DataFrameIO.cs

```diff
@@ -0,0 +1,266 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
This file is mostly a duplicate of M.D.A.DataFrameIO.cs. I've called out the minor changes as comments